tests/baselines/reference/contextualTypeAny.errors.txt

```diff
@@ -0,0 +1,15 @@
+tests/cases/compiler/contextualTypeAny.ts(3,5): error TS2322: Type '{ p: string; q: any; }' is not assignable to type '{ [s: string]: number; }'.
+  Property 'p' is incompatible with index signature.

```

Comment:
Can you not report the index signature here? This is probably confusing for users.